README.md

```diff
@@ -1,89 +1,96 @@
-# Stripe Python bindings [![Build Status](https://travis-ci.org/stripe/stripe-python.svg?branch=master)](https://travis-ci.org/stripe/stripe-python)
+# Stripe Python Library [![Build Status](https://travis-ci.org/stripe/stripe-python.svg?branch=master)](https://travis-ci.org/stripe/stripe-python)
+
+The Stripe Python library provides convenient access to the Stripe API from
+applications written in the Python language. It includes a pre-defined set of
+classes for API resources that initialize themselves dynamically from API
+responses which makes it compatible with a wide range of versions of the Stripe
+API.
+
+## Documentation
+
+See the [Python API docs](https://stripe.com/docs/api/python#intro).
 
 ## Installation
 
-You don't need this source code unless you want to modify the
-package. If you just want to use the Stripe Python bindings, you
-should run:
+You don't need this source code unless you want to modify the gem. If you just
+want to use the package, just run:
 
     pip install --upgrade stripe
 
 or
 
     easy_install --upgrade stripe
 
-See http://www.pip-installer.org/en/latest/index.html for instructions
-on installing pip. If you are on a system with easy_install but not
-pip, you can use easy_install instead. If you're not using virtualenv,
-you may have to prefix those commands with `sudo`. You can learn more
-about virtualenv at http://www.virtualenv.org/
-
-To install from source, run:
+Install from source with:
 
     python setup.py install
 
-## Documentation
+### Requirements
+
+* Python 2.6+ or Python 3.3+ (PyPy supported)
 
```

Comment:
It's not a requirement, but maybe we should briefly mention that the library supports multiple HTTP clients and that `requests` is the recommended choice?